tests/test_wave_01.py

```diff
@@ -175,20 +172,29 @@ def test_moves_movie_from_watchlist_to_watched():
         ],
         "watched": [FANTASY_2]
     }
+    print("Before watch_movie:")
+    print("Watchlist:", janes_data["watchlist"])
+    print("Watched:", janes_data["watched"])
 
     # Act
     updated_data = watch_movie(janes_data, movie_to_watch["title"])
 
+    print("After watch_movie:")
+    print("Watchlist:", updated_data["watchlist"])
+    print("Watched:", updated_data["watched"])
+
     # Assert
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
+    assert HORROR_1 == updated_data["watched"][1]["title"]
```

Comment:
```suggestion
    assert HORROR_1["title"] == updated_data["watched"][1]["title"]
```

After looking at the definition of `HORROR_1` in `tests/test_constants.py`, I realized that this change passes the tests!